bin/zopen-importenvs

```diff
@@ -1,18 +1,87 @@
-#!/bin/sh
-#
+#!/bin/bash
 # Used for development to source all the boot .envs
-# . importenvs
+#. ./zopen-importenvs <path to buildenv to fetch dependency>
 
-if [ ! -d "${HOME}/zopen/boot/" ]; then
```

Comment:
I was just thinking that this script could be misused - I can see users trying to calling it like ./zopen-importenvs instead of sourcing it.

We should have a way to detect that if it's not being sourced, print out an error message.

You can use the code from here: https://stackoverflow.com/questions/2683279/how-to-detect-if-a-script-is-being-sourced

Reply:
+1